packages/paste-website/src/components/site-wrapper/site-header/DarkModeToggle.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import {trackCustomEvent} from 'gatsby-plugin-google-analytics';
+import {styled, css} from '@twilio-paste/styling-library';
+import {useUID} from '@twilio-paste/uid-library';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {Box} from '@twilio-paste/box';
+import {Stack} from '@twilio-paste/stack';
+import {DarkModeIcon} from '@twilio-paste/icons/esm/DarkModeIcon';
+import {LightModeIcon} from '@twilio-paste/icons/esm/LightModeIcon';
+import {useDarkModeContext} from '../../../context/DarkModeContext';
+
+const StyledThemeSwitcherLabel = styled.label<{checked: boolean}>((props) =>
+  css({
+    cursor: 'pointer',
+    display: 'inline-block',
+    flexGrow: 1,
+    justifyContent: 'center',
+    fontSize: 'fontSize30',
+    fontWeight: props.checked ? 'fontWeightBold' : 'fontWeightNormal',
+    lineHeight: 'lineHeight30',
+    paddingTop: 'space20',
+    paddingBottom: 'space20',
+    paddingRight: 'space40',
+    paddingLeft: 'space40',
+    backgroundColor: props.checked ? 'colorBackgroundStrongest' : 'transparent',
+    borderRadius: 'borderRadius10',
+    color: props.checked ? 'colorTextWeakest' : 'colorTextInverse',
+    '&:hover': {
+      textDecoration: 'underline',
+    },
+    '&:focus-within': {
+      boxShadow: 'shadowFocusInverse',
+      textDecoration: 'underline',
+    },
+  })
+);
+
+const StyledThemeSwitcherRadio = styled.input({
+  opacity: 0,
+  position: 'absolute',
+});
+
+export const DarkModeToggle: React.FC = () => {
+  const {theme, toggleMode} = useDarkModeContext();
+  const consoleID = useUID();
+  const pasteID = useUID();
```

Comment:
We should change these to `defaultID` and `darkID`. Or maybe `lightID` and `darkID`. Something like that.

Reply:
Probably